src/indexers/vaults-indexer.ts

```diff
@@ -17,8 +17,9 @@ export async function indexProtocolVaults() {
         vaults.map(async (vault) => {
           try {
             const snapshot = await vaultToSnapshot(chain, vault);
-            await mapper.put(Object.assign(new HistoricVaultSnapshotOldModel(), snapshot));
-            if (!vault.isMigrating) {
+            if (vault.isMigrating) {
```

Comment:
wouldn't this cause old charts to not be accurate anymore? making us unable to revert this migration